features/google/src/components/importExplanationExpandable/importExplanationExpandable.css

```diff
@@ -0,0 +1,157 @@
+.import-view .explanation-expandable {
+    flex: 1 1 auto;
+    display: flex;
+    flex-direction: column;
+    color: var(--text-dark);
+    overflow: hidden scroll;
+    margin: 0 auto;
+}
+
+.import-view .explanation-expandable .intro {
+    width: 100%;
+    max-width: var(--max-width);
+    padding: 0 16px 16px 16px;
+    box-sizing: border-box;
+    margin: 4px auto 72px auto;
+}
+
+.import-view .explanation-expandable .intro p {
+    font-family: "Jost Bold";
+    font-size: 40px;
+    line-height: var(--line-height);
+    letter-spacing: -0.01em;
+    margin: 0 0 16px 0;
+}
+
+.import-view .explanation-expandable .head {
+    width: 100%;
+    max-width: var(--max-width);
+    height: 96px;
+    border-top: 1px solid var(--color-gray-400);
+    overflow: hidden;
+    display: flex;
+    align-items: center;
+    padding: 24px;
+    box-sizing: border-box;
+}
+
+.import-view .explanation-expandable .head .number {
+    width: 48px;
+    height: 48px;
+    border-radius: 50%;
+    font-size: 28px;
+    font-weight: 600;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--color-indigo-950);
+    color: var(--text-light);
+}
+
+.import-view .explanation-expandable .head .heading {
+    font-family: "Jost Medium";
+    margin-left: 18px;
+    font-size: 24px;
+    flex: 1 1 auto;
+}
+
+.import-view .explanation-expandable .head .rotate-180 {
+    transform: rotate(180deg);
+}
+
+.import-view .explanation-expandable .section-body {
+    padding: 16px 16px 16px 16px;
+    width: 100%;
+    max-width: var(--max-width);
+    box-sizing: border-box;
+    border-top: 1px solid var(--color-gray-400);
+}
+
+.import-view .explanation-expandable .section-body .full-screen {
+    width: 100%;
+    max-width: var(--max-width);
+    margin-bottom: 25px;
+}
+
+.import-view .explanation-expandable .section-body .file-info p {
+    font-size: 16px;
+    color: var(--color-indigo-950);
+    margin: 0 0 2px 0;
+    line-height: 120%;
+}
+
+.import-view .explanation-expandable .section-body .x-divider {
+    display: flex;
+    flex-direction: row;
+    margin: 0;
+}
+
+.import-view .explanation-expandable .separator {
+    height: 1px;
+    width: 100%;
+    max-width: var(--max-width);
+    background-color: var(--color-gray-400);
+    margin-bottom: 26px;
+}
+
+.import-view .explanation-expandable .translucent-sides {
```

Comment:
naming nit: translucent is a bit less than transparent, and this seems to be totally transparent, right?